tests/torch/data/reference_graphs/quantized/adjust_paddings/vpu_bd_int8.dot

```diff
@@ -1,20 +1,20 @@
 strict digraph  {
 "0 /nncf_model_input_0" [id=0, label="nncf_model_input_#0", style=filled, type=nncf_model_input];
 "1 MultiBranchesModel/ReLU/relu_0" [id=1, label="relu_#1", style=filled, type=relu];
-"2 MultiBranchesModel/NNCFConv2d[conv_a]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=2, label="WFQ_[B:4 M:S SGN:U PC:Y]_#2_G0", style=filled, type=symmetric_quantize];
+"2 MultiBranchesModel/NNCFConv2d[conv_a]/ModuleDict[pre_ops]/UpdateWeight[1]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=2, label="WFQ_[B:4 M:S SGN:U PC:Y]_#2_G0", style=filled, type=symmetric_quantize];
 "3 MultiBranchesModel/NNCFConv2d[conv_a]/SymmetricQuantizer/symmetric_quantize_0" [color=green, id=3, label="AFQ_[B:8 M:S SGN:U PC:Y]_#3_G0", style=filled, type=symmetric_quantize];
 "4 MultiBranchesModel/NNCFConv2d[conv_a]/conv2d_0" [color=purple, id=4, label="DW_Conv2d_k3x3_PAD_#4", style=filled, type=conv2d];
 "5 MultiBranchesModel/MaxPool2d[max_pool_b]/AsymmetricQuantizer/asymmetric_quantize_0" [color=green, id=5, label="AFQ_[B:8 M:A SGN:S PC:N]_#5_G1", style=filled, type=asymmetric_quantize];
 "6 MultiBranchesModel/MaxPool2d[max_pool_b]/max_pool2d_0" [id=6, label="max_pool2d_#6", style=filled, type=max_pool2d];
-"7 MultiBranchesModel/NNCFConv2d[conv_b]/ModuleDict[pre_ops]/UpdateWeight[0]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=7, label="WFQ_[B:4 M:S SGN:U PC:Y]_#7_G1", style=filled, type=symmetric_quantize];
+"7 MultiBranchesModel/NNCFConv2d[conv_b]/ModuleDict[pre_ops]/UpdateWeight[1]/SymmetricQuantizer[op]/symmetric_quantize_0" [color=red, id=7, label="WFQ_[B:4 M:S SGN:U PC:Y]_#7_G1", style=filled, type=symmetric_quantize];
```

Comment:
@ljaljushkin, is it ok if references will be containing the index of op insertion? This should be reproducible for all test cases

Reply:
As I found, this counter should be at least thread local because test `est_autoq_precision_init/test_can_broadcast_initialized_precisions_in_distributed_mode/` requires this